README.md

```diff
@@ -243,6 +244,67 @@ For more information and how to use your extension in production see the documen
 [opentok/screensharing-extensions](https://github.com/opentok/screensharing-extensions/blob/master/chrome/ScreenSharing/README.md#customizing-the-extension-for-your-website)
 repo on GitHub.
 
+### Phone dial-out
```

Comment:
The info looks nice and clear.
However, are we following the philosophy of separating detailed readme components like this into separate files? (I'm thinking of the INSTALL-heroku.md file).